docs/architecture/blazor-for-web-forms-developers/project-structure.md

```diff
@@ -1,9 +1,230 @@
 ---
-title: Project structure for ASP.NET Web Forms and Blazor apps
+title: Project structure for Blazor apps
 description: Comparing the project structure of ASP.NET Web Forms and Blazor projects.
 author: danroth27
 ms.author: daroth
 ms.date: 09/11/2019
 ---
 
-# Project structure for ASP.NET Web Forms and Blazor apps
+# Project structure for Blazor apps
+
+Blazor apps share a lot in common with ASP.NET Web Froms apps. Even though the projects might look quite different, the concepts have many similarities. Here we will look at the structure of a Blazor project and compare it to ASP.NET Web Forms projects.
+
+To create your first Blazor app follow the instructions in the [Blazor getting started steps](https://docs.microsoft.com/aspnet/core/blazor/get-started). You can follow the instructions to create either a Blazor Server app or a Blazor WebAssembly app hosted in ASP.NET Core. Most of the code in both projects is the same, although the hosting model specific logic will differ.
+
+## Project file
+
+Blazor Server apps are .NET Core projects. The project file for the Blazor Server app is about as simple as it can get:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+</Project>
+```
+
+The project file for a Blazor WebAssembly looks slightly more involved (exact version numbers may vary):
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <OutputType>Exe</OutputType>
```

Comment:
This looks weird - it's an `exe` type but targeting `netstandard`? You explain the `netstandard` below, but maybe comment about the `exe` since usually `netstandard` is recommended for libs

Reply:
Yeah, I'm not actually sure why we set the OutputType to Exe. Things seem to work without it. I'll find out. The tricky thing here is that the Blazor WebAssembly details are likely to change over the upcoming months we shift focus to productizing it.

Reply:
Makes sense.